synapse_spamcheck_badlist/bad_list_filter.py

```diff
@@ -120,7 +120,8 @@ async def check_event_for_spam(self, event) -> bool:
             return False
 
         # If it's a text content, look for links.
-        if event["content"]["msgtype"] == "m.text":
+        # Note that even files have a text content (the description).
+        if event["content"]["msgtype"] in ["m.text", "m.emote", "m.notice", "m.file", "m.image", "m.audio"]:
```

Comment:
all `m.room.message` events should have a `body` regardless of `msgtype`

Reply:
Thanks. Now filtering all instances of `m.room.message`s